internal/avro/avro.go

```diff
@@ -0,0 +1,36 @@
+package avro
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Schema struct {
+	// Schema is the Avro schema string.
+	Schema string `json:"schema"`
+	// Subject where the schema is registered for.
+	Subject string `json:"subject"`
+	// Version of the returned schema.
+	Version int `json:"version"`
+	ID      int `json:"id,omitempty"`
+}
+
+func GetSchema(avroRegistry string) (string, int) {
```

Comment:
we'll likely want a way to dependency inject the http client here. Things like TLS support and the like seem pertinent.